pages/docs/installation/plugins.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Grouparoo Plugins"
+date: "2021-02-10"
+section: "installation"
+pullQuote: "Grouparoo can be extended with Plugins to add more Apps, Sources, and Destinations."
+---
+
+Grouparoo can be extended with Plugins to add more Apps, Sources, and Destinations.
+
+## Installing a Plugin
+
+The Grouparoo command line tool is used to add a plugin to your project via `grouparoo install [package]`. You can learn more about the `grouparoo install` command [here](/docs/cli/setup). So, if you wanted to install the `@grouparoo/mysql` plugin, you would run:
+
+```bash
+grouparoo install @grouparoo/mysql # install the latest version
+```
+
+That's it!
+
+Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly**. If you need to install a specific version of a plugin to match `@grouparoo/core`, you can:
+
+```bash
+grouparoo install @grouparoo/mysql@v1.2.3 # install v1.2.3 specifically
+```
+
+## Available Community Plugins
+
+<PluginsList />
+
+## Paid Plugins
+
+While many Grouparoo plugins are available for free via an open-source license, some require a commercial license.
+
+<Alert variant="primary">
+  If you are interested in obtaining a license to our commercial plugins, please{" "}
+  <a href="/meet">contact us</a>.
```

Comment:
Can you include some ref params for tracking?

Reply:
I added them, but what do you expect those URL params to do?  It doesn't look like we send them though the form?  We know that lots of folks block google analytics, and sometimes to next a route changes is not a page change that Google Analytics will pick up on...